README.md

```diff
@@ -60,9 +60,9 @@ pip install pyronear
 
 ## Usage
 
-### datasets
+### Python package
 
-Access all pyrovision datasets just like any `torchvision.datasets.VisionDataset`:
+You can use the library like any other python package to analyze your documents as follows:
```

Comment:
Why did you removed this here ?

Reply:
I was starting to think that this is the purpose of the documentation, not the README. Usage instructions should be plain user-friendly examples of how to use the package in my opinion.

Reply:
ok ok